ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -26,6 +23,8 @@ QtObject {
     property bool addingSavedAddress: false
     property bool deletingSavedAddress: false
 
+    readonly property SharedStores.NetworksStore networksStore: SharedStores.NetworksStore {}
```

Comment:
Oops, nice catch, thanks!
The comment `Required until tokensStore, walletAssetsStore and currentActivityFiltersStore are reworked` still stands though, we want to avoid these singleton stores in favour of modular ones with explicit dependency injection